### PR TITLE
Feature/page authorities visualization management 860q6tyq2

### PR DESCRIPTION
**Pull request by theghost1980**

### CODE REVIEW

File: src/popup/pages/app-container/settings/accounts/manage-account-authorities/manage-account-authorities.component.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@import '../../../../../variable.scss';
+
+.settings-manage-account-authorities {
+  @include pageLayout;
+  justify-content: flex-start;
+  overflow-x: hidden;
```
Comment (cedricguillas) [inline src/popup/pages/app-container/settings/accounts/manage-account-authorities/manage-account-authorities.component.scss:6]: why do we need to specify overflow-x ?

**Review by cedricguillas — COMMENTED** [review]
please check the comments

File: src/popup/pages/app-container/settings/accounts/manage-account-authorities/manage-account-authorities.component.scss (hunk 1 above)
Comment (theghost1980) [reply src/popup/pages/app-container/settings/accounts/manage-account-authorities/manage-account-authorities.component.scss:6]: removed :)

**Review by stoodkev — CHANGES_REQUESTED** [review]
Please change the following : 
- Remove weight and threshold. Having there is weird since we only put the authorities and not the keys.
- Change `Key` by `Authority` in `Posting/Active Key`
- If there's no Posting Authority hide the section, same for active
- If neither posting or active authority : "No authority has been given to other accounts"

**Review by cedricguillas — APPROVED** [review]

**Pull request merged**